docs/manual/developer_guide.adoc

```diff
@@ -1337,13 +1337,321 @@ Available low-level Jinja macros that can be used in Bash remediations:
+** *file_regex* - Regular expression that matches file names in a directory specified by *filepath*. Can be set only if *filepath* parameter specifies a directory.
+** *fileuid* - user ID (UID)
+* Languages: Ansible, Bash, OVAL
+
+file_permissions::
+* Checks permissions (mode) on a given file.
+* Parameters:
+** *filepath* - File path to be checked. If the file path ends with `/` it describes a directory.
+** *file_regex* - Regular expression that matches file names in a directory specified by *filepath*. Can be set only if *filepath* parameter specifies a directory.
+** *filemode* - File permissions in a hexadecimal format, eg. `'0640'`.
+* Languages: Ansible, Bash, OVAL
+
+grub2_bootloader_argument::
+* Checks kernel command line arguments in GRUB 2 configuration.
+* Parameters:
+** *arg_name* - argument name, eg. `audit`
+** *arg_value* - argument value, eg. `'1'`
+* Languages: Ansible, Bash, OVAL
+
+kernel_module_disabled::
+* Checks if the given Linux kernel module is disabled.
+* Parameters:
+** *kernmodule* - name of the Linux kernel module, eg. `cramfs`
+* Languages: Ansible, Bash, OVAL
+
+mount::
+* Checks that a given mount point is located on a separate partition.
+* Parameters:
+** *mountpoint* - path to the mount point, eg. `/var/tmp`
+* Languages: Anaconda, OVAL
+
+mount_option::
+* Checks if a given partition is mounted with a specific option such as "nosuid".
+* Parameters:
+** *mountpoint* - mount point on the filesystem eg. `/dev/shm`
+** *mountoption* - mount option, eg. `nosuid`
+** *filesystem* - filesystem in `/etc/fstab`, eg. `tmpfs`. Used only in Bash remediation.
+** *type* - filesystem type. Used only in Bash remediation.
+** *mount_has_to_exist* - Used only in Bash remediation. Specifies if the *mountpoint* entry has to exist in `/etc/fstab`. If set to `yes` and the *mountpoint* entry is not present in `/etc/fstab` the Bash remediation terminates.
+* Languages: Anaconda, Ansible, Bash, OVAL
+
+mount_option_remote_filesystems::
+* Checks if all remote filesystems (NFS mounts in `/etc/fstab`) are mounted with a specific option.
+* Parameters:
+** *mountpoint* - always set to `remote_filesystems`
+** *mountoption* - mount option, eg. `nodev`
+** *filesystem* - filesystem of new mount point (used when adding new entry in `/etc/fstab`), eg. `tmpfs`. Used only in Bash remediation.
+* Languages: Ansible, Bash, OVAL
+
+mount_option_removable_partitions::
+* Checks if all removable media mounts are mounted with a specific option.
+* Parameters:
+** *mountpoint* - always set to `remote_filesystems`
+** *mountoption* - mount option, eg. `nodev`
+** *filesystem* - filesystem of new mount point (used when adding new entry in `/etc/fstab`), eg. `tmpfs`. Used only in Bash remediation.
+* Languages: Anaconda, Ansible, Bash, OVAL
+
+package_installed::
+* Checks if a given package is installed. Optionally, it can also check whether a specific version or newer is installed.
```

Comment:
In the parameters below, the evr parameter is said to check for specific value. How to specify that we want newer version, eg greater or equal to the evr?

Reply:
Actually, it checks if the given version or a newer version is installed, because OVAL check uses  operation "greater than or equal".    It's here:  https://github.com/ComplianceAsCode/content/blob/85bdd8bbb8753f77f6c728811c89f026b76b4d8b/shared/macros-oval.jinja#L371

At this moment it isn't possible to parametrize the operation, eg. to check for older version or for the exactly the same version.


I'll try to improve the wording.